### PR TITLE
fix(utils): Omits undefined values from serialized params

### PR DESCRIPTION
**Pull request by markwaddle**
Undefined is a valid value in an object, but not in a parameter. Therefore it would be helpful if `convertObjectToString` simply omitted them. #28

### CODE REVIEW

File: src/angularODataUtils.ts [hunk 1]
```diff
@@ -3,8 +3,8 @@ export class ODataUtils {
         const properties: string[] = [];
 
         for (const prop in obj) {
-            if (obj.hasOwnProperty(prop)) {
-                const value: string = this.quoteValue(obj[prop]);
+            if (obj.hasOwnProperty(prop) && obj[prop] !== undefined) {
```
Comment (StefH) [inline src/angularODataUtils.ts]: It this check enough? Or do you also need to check on `null` ?

File: test/angularODataUtils.spec.ts [hunk 2]
```diff
@@ -37,7 +37,8 @@ describe('ODataUtils', () => {
 
     it('convertObjectToString', () => {
         // Act
-        const value: string = ODataUtils.convertObjectToString({ str: 'abc', int: 10, double: -10.01, guid: 'eefea99a-c988-44b8-ac37-b326a489c1e3' });
+        const value: string = ODataUtils.convertObjectToString(
+          { str: 'abc', int: 10, double: -10.01, guid: 'eefea99a-c988-44b8-ac37-b326a489c1e3', omitted: undefined });
```
Comment (StefH) [inline test/angularODataUtils.spec.ts:41]: also add a test for null?
like `nullvalue: null` ?

**Review by StefH — CHANGES_REQUESTED** [review]
Thanks. Can you please take a look at my question?

File: src/angularODataUtils.ts (hunk 1 above)
Comment (Sir-J) [inline src/angularODataUtils.ts]: May be ``if (obj.hasOwnProperty(prop) && obj[prop])``?

Comment (Sir-J) [reply src/angularODataUtils.ts]: I think it's enough. Null need send to server.

**Review by StefH — APPROVED** [review]
